RealestateApi/Controllers/RealEstateController.cs

```diff
@@ -21,7 +21,7 @@ public RealEstateController(ILogger<RealEstateController> logger, IRealEstateSer
         }
 
         [HttpGet(Name = "GetAllRealEstate")]
-        public async Task<List<RealEstate>> GetAllRealEstate()
+        public async Task<List<RealEstateMoodel>> GetAllRealEstate()
```

Comment:
Didn't you mean to write "RealEstateModel" instead of "RealEstateMoodel" ? If yes you should correct in the other files aswell.